db/data.py

```diff
@@ -424,31 +422,54 @@ def load_many_low_level(recordings):
         return dict(recordings_info)
 
 
-def load_high_level(mbid, offset=0):
+def map_highlevel_class_names(highlevel, mapping):
+    """Convert class names from the classifier output to human readable names.
+
+    Arguments:
+        highlevel (dict): highlevel data dict containing shortend keys
+        mapping (dict): a mapping from class names -> human readable names
+
+    Returns:
+        the highlevel input with the keys of the `all` item, and the `value` item
+        changed to the values from the provided mapping
+    """
+
+    new_all = {}
+    for cl, val in highlevel["all"].items():
+        new_all[mapping[cl]] = val
```

Comment:
this assumes that the mapping has all possible values in it. Since we control the both the models and the mapping definition I didn't bother checking it here